includes/abstracts/llms.abstract.notification.controller.php

```diff
@@ -457,7 +458,7 @@ public function has_subscriber_received( $type, $subscriber ) {
 			)
 		);
 
-		return $query->found_results ? true : false;
+		return (bool) $query->get_found_results();
```

Comment:
```suggestion
		return $query->has_results();
```

I think that this maybe semantically doesn't matter that much but this function already exists

Reply:
The result is the same but it is semantically better. Thanks!

Reply:
I wish I had given that commit a better message than the default on that GitHub provided.